src/adminApp/OrgManager.js

```diff
@@ -50,6 +51,9 @@ class OrgManager extends Component {
   getChildContext() {
     return { store };
   }
+  shouldComponentUpdate(nextProps, nextState, nextContext) {
+    return !_.isEqual(_.omit(nextProps, ["location"]), _.omit(this.props, ["location"]));
```

Comment:
Due to performance considerations, it would be better if we did a shallow compare within  shouldComponentUpdate().